flaskapp/forms.py

```diff
@@ -12,3 +12,8 @@ class QuestionForm(FlaskForm):
                               validators=[DataRequired(), NumberRange(1, 100, "Not a valid difficulty")])
     imp = BooleanField('imp')
     submit = SubmitField('submit')
+    option1 = TextAreaField('Option1')
```

Comment:
It shouldn't be like this there should be two forms one for subjective and one for objective. Make a form with selector to select type of the question then provide form with appropriate type